### DISCUSSION TITLE
Keyboard navigation

### DISCUSSION THREAD

**Discussion by Abhimishra2424** [KevinVandy/material-react-table#313] [General]
plz, add **Keyboard navigation** Use the arrow keys to move the focus. , and select rows when we do key up and down

**Comment by KevinVandy** [comment]
You can add your own event listeners and styles to all of the components. This library is not meant to be an excel replacement like AG grid, but you can customize it further for your needs
https://www.material-react-table.com/docs/guides/table-event-listeners
